arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,595 @@
+          },
+          {
+            "name": "containerServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the container service within the given subscription and resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "Not found"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/containerServices": {
+      "get": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_ListByResourceGroup",
+        "description": "The operation to list container services.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerServiceListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+    },
+    "definitions": {
+        "Resource": {
+          "description": "The Resource model definition.",
+          "properties": {
+            "id": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource Id"
+            },
+            "name": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource name"
+            },
+            "type": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource type"
+            },
+            "location": {
+              "type": "string",
+              "description": "Resource location"
+            },
+            "tags": {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "description": "Resource tags"
+            }
+          },
+          "required": [
+            "location"
+          ],
+          "x-ms-azure-resource": true
+        },
+        "ContainerServiceCustomProfile": {
+          "properties": {
+            "orchestrator": {
+              "type": "string",
```

Comment:
Do you want to add a regex on the naming pattern of the orchestrator? Is there a min length/ max length constraint on it? Please make sure that all such constraints are accurately modeled in the spec wherever necessary.